configure.ac

```diff
@@ -286,6 +286,8 @@ echo $gui
 echo
 changequote([,])
 
+AC_DEFINE([GVARS3_HAVE_TOON], [], [Description])
```

Comment:
these ought to be defined, though you need to have TooN installed and accessible before running `./configure` for gvars

Reply:
I did try to remove that patch, but then I get an error when the build system runs "autoheader", see extract from the build log:
```
dh binary
   dh_update_autotools_config
   dh_autoreconf
configure.ac:70: warning: AC_LANG_CONFTEST: no AC_LANG_SOURCE call detected in body
./lib/autoconf/lang.m4:199: AC_LANG_CONFTEST is expanded from...
./lib/autoconf/general.m4:2823: _AC_COMPILE_IFELSE is expanded from...
./lib/autoconf/general.m4:2839: AC_COMPILE_IFELSE is expanded from...
configure.ac:70: the top level
configure.ac:71: warning: AC_LANG_CONFTEST: no AC_LANG_SOURCE call detected in body
./lib/autoconf/lang.m4:199: AC_LANG_CONFTEST is expanded from...
./lib/autoconf/general.m4:2823: _AC_COMPILE_IFELSE is expanded from...
./lib/autoconf/general.m4:2839: AC_COMPILE_IFELSE is expanded from...
configure.ac:71: the top level
configure.ac:92: warning: AC_LANG_CONFTEST: no AC_LANG_SOURCE call detected in body
./lib/autoconf/lang.m4:199: AC_LANG_CONFTEST is expanded from...
./lib/autoconf/general.m4:2823: _AC_COMPILE_IFELSE is expanded from...
./lib/autoconf/general.m4:2839: AC_COMPILE_IFELSE is expanded from...
configure.ac:92: the top level
configure.ac:108: warning: AC_LANG_CONFTEST: no AC_LANG_SOURCE call detected in body
./lib/autoconf/lang.m4:199: AC_LANG_CONFTEST is expanded from...
./lib/autoconf/general.m4:2823: _AC_COMPILE_IFELSE is expanded from...
./lib/autoconf/general.m4:2839: AC_COMPILE_IFELSE is expanded from...
configure.ac:108: the top level
configure.ac:120: warning: AC_LANG_CONFTEST: no AC_LANG_SOURCE call detected in body
./lib/autoconf/lang.m4:199: AC_LANG_CONFTEST is expanded from...
./lib/autoconf/general.m4:2823: _AC_COMPILE_IFELSE is expanded from...
./lib/autoconf/general.m4:2839: AC_COMPILE_IFELSE is expanded from...
configure.ac:120: the top level
configure.ac:291: warning: AC_OUTPUT should be used without arguments.
configure.ac:291: You should run autoupdate.
autoheader: warning: missing template: GVARS3_HAVE_TOON
autoheader: warning: Use AC_DEFINE([GVARS3_HAVE_TOON], [], [Description])
autoheader: warning: missing template: GVARS3_HAVE_TOON2
autoreconf: error: /usr/bin/autoheader failed with exit status: 1
dh_autoreconf: error: autoreconf -f -i returned exit code 1
make: *** [debian/rules:4: binary] Error 25
```
So even though I can of course keep a Debian-specific patch, I stand by my merge request :-)

Reply:
OK, I'll look into it. I think this is to do with some of the more advanced integration for automake? I've not used automake here, only autoconf, so the usual way of running is:
```
autoconf
./configure
```